composer.json

```diff
@@ -41,17 +41,13 @@
             },
```

Comment:
@e0ipso  The only way I could get it to work was as suggested by @cjgratacos by pointing to a branch with an updated composer.json. Unless there's another way?